core/chains/evm/types/types.go

```diff
@@ -117,7 +117,7 @@ func (c *ChainCfg) Value() (driver.Value, error) {
 
 type DBChain = chains.DBChain[utils.Big, *ChainCfg]
 
-// https://app.shortcut.com/chainlinklabs/story/33622/remove-legacy-config
+// TODO: https://app.shortcut.com/chainlinklabs/story/33622/remove-legacy-config
```

Comment:
@jmank88 Is this comment still relevant?

Reply:
Yes. This type will at least need to be updated, and may not be used at all after removing legacy config.